### PR TITLE
Add (and run) prettier sort imports plugin

### PR DESCRIPTION
**Pull request by raphaelts3**
Closes https://github.com/nerdyman/react-compare-slider/issues/84

Hey @nerdyman, I've used the configurations provided [here](https://github.com/IanVS/prettier-plugin-sort-imports#usage), except for those already present in the config, please, let me know if I should change something and re-run.

### CODE REVIEW

**Review by nerdyman — CHANGES_REQUESTED** [review]
Thanks for your contribution! Just a really minor tweak and this is good to go.

File: package.json [hunk 1]
```diff
@@ -87,7 +87,22 @@
     "printWidth": 90,
     "semi": true,
     "singleQuote": true,
-    "trailingComma": "es5"
+    "trailingComma": "es5",
+    "importOrder": [
+      "^@core/(.*)$",
```
[the remark this answers is not in the document]
Comment (raphaelts3) [reply package.json]: Adjusted!

**Comment by nerdyman** [comment]
Thanks for sorting this @raphaelts3! Can you drop a comment in the issue too so I can assign it to you? Don't want you to miss out on Hacktober points 😃 

https://github.com/nerdyman/react-compare-slider/issues/84

**Review by nerdyman — APPROVED** [review]
LGTM 🚀

**Pull request merged**